content/docs/concepts/architecture/_index.md

```diff
@@ -104,7 +145,69 @@ When distribution occurs:
 
 To learn more about the concept see [Staking rewards and distribution](/concepts/staking/#staking-rewards-and-distribution) and [Staking accounts](/concepts/staking/#staking-accounts).
 
-## Blockchain Consensus
+### Autonity Accountability Contract
+The contract implementing the accountability and fault detection (AFD) protocol extensions, including primitives for misbehaviour accusations, proving innocence against an accusation, proven faults, slashing, and jailing.
+
+The contract stores static [slashing protocol configuration parameters](/concepts/accountability/#slashing-protocol-configuration) used to compute slashing penalties.
+
+Contract functions for returning a committee member's proven faults, if a new accusation can be made and is slashable, and to submit accountability events are called by validators whilst participating in the AFD protocol. Function calls to compute accountability each block and apply slashing penalties at epoch end are restricted to protocol.
```

Comment:
I think the purpose of this paragraph would be to highlight which functions are called by the protocol and which by single validator. If yes, I would rephrase it as it is a bit hard to parse (at least for me). Maybe using a bulleted list would be more efficient.

Reply:
Yes. Edited and converted to a list. Commit f0abfb26c62401bfb3d3de49897a639a5944840c.

Reply:
Can this be resolved after the edit made @lorenzo-dev1 ?